Blazorade.Bootstrap.Components/Blazorade.Bootstrap.Components.csproj

```diff
@@ -19,8 +19,9 @@
 
 
   <ItemGroup>
-    <PackageReference Include="Blazorade.Core" Version="0.9.0-preview" />
+    <PackageReference Include="Blazorade.Core" Version="0.9.0.2-preview" />
```

Comment:
This is updated in the main repo as well. It's always a good practice to get the latest versions from master to your development branch before starting to merge the dev branch back to the master branch.